rotkehlchen/chain/ethereum/airdrops.py

```diff
@@ -89,7 +90,7 @@
         A_CVX,
         'https://www.convexfinance.com/',
     ),
-    'fox': (
+    'shapeshift': (
```

Comment:
so this change is somehow messing with the icon on the airdrops page. The DIVA one works fine.
![2023-08-08_22-08](https://github.com/rotki/rotki/assets/1658405/0f70000c-d8cc-4477-9abe-a5bba67cdcb1)